parsl/dataflow/dflow.py

```diff
@@ -890,7 +1001,8 @@ def check_dep(d: Any) -> None:
             check_dep(dep)
 
         # Check for futures in inputs=[<fut>...]
-        for dep in kwargs.get('inputs', []):
+        inp = kwargs.get('inputs', [])
```

Comment:
this doesn't change any behaviour, right?

Reply:
I do not believe so, but I reverted it anyway. I believe I changed it for the dynamic files stuff, which will be a seperate PR, but the code was pulled from this one.